### DISCUSSION TITLE
JDM parsing

### DISCUSSION THREAD

**Discussion by slice-Adityaraj** [gorules/zen-go#27] [Q&A]
Is there any way I can get all the variables required for a JDM to execute completely?

**Comment by ivanmiletic** [comment]
Do you mean, what is required input model for the given graph?

**Reply by slice-Adityaraj** [reply]
Apologies for the delay in response.
Yes that's what I meant, is there a way I can tell a user from the JDM model the list of inputs that he needs to provide?
